### PR TITLE
GPDB DOCS - XML gucs xmlbinary, xmloption

### PR DESCRIPTION
**Pull request by mkiyama**
Add XML GUCs xmlbinary, xmloption.
Also added note to COPY about copying XML data.

The GUC reference information was reviewed previously.

[ci skip]

### CODE REVIEW

File: gpdb-doc/dita/ref_guide/config_params/guc-list.xml [hunk 1]
```diff
@@ -9082,4 +9086,77 @@ $ gpconfig -c pgcrypto.fips -v on --masteronly</codeblock></p>
       </table>
     </body>
   </topic>
+  <topic id="xmlbinary">
+    <title>xmlbinary</title>
+    <body>
+      <p>Specifies how binary values are encoded in XML data. For example, when
+          <codeph>bytea</codeph> values are converted to XML. The binary data can be converted to
+        either base64 encoding or hexadecimal encoding. The default is base64.</p>
+      <p>The parameter can be set for a database system, an individual database or a session.</p>
```
Comment (dyozie) [inline gpdb-doc/dita/ref_guide/config_params/guc-list.xml]: \+ Oxford comma

File: gpdb-doc/dita/ref_guide/config_params/guc-list.xml [hunk 2]
```diff
@@ -9082,4 +9086,77 @@ $ gpconfig -c pgcrypto.fips -v on --masteronly</codeblock></p>
...
+  <topic id="xmloption">
+    <title> xmloption</title>
+    <body>
+      <p>Specifies whether XML data is to be considered as an XML document
+          (<codeph>document</codeph>) or XML content fragment (<codeph>content</codeph>) for
+        operations that perform implicit parsing and serialization. The default is
+          <codeph>content</codeph>. </p>
+      <p>This parameter affects the validation performed by <codeph>xml_is_well_formed()</codeph>.
+        If the value is <codeph>document</codeph>, the function checks for a well-formed XML
+        document, If the value is <codeph>content</codeph>, the function checks for a well-formed
```
Comment (dyozie) [inline gpdb-doc/dita/ref_guide/config_params/guc-list.xml]: Change "document, If" -> "document. If"

File: gpdb-doc/dita/ref_guide/config_params/guc-list.xml [hunk 3]
```diff
@@ -9082,4 +9086,77 @@ $ gpconfig -c pgcrypto.fips -v on --masteronly</codeblock></p>
...
+        XML content fragment.</p>
+      <note>An XML document that contains a document type declaration (DTD) is not considered a
+        valid XML content fragment. If <codeph>xmloption</codeph> set to <codeph>content</codeph>,
+        XML that contains a DTD is not considered valid XML.<p>To cast a character string that
+          contains a DTD to <codeph>xml</codeph> data type, use the <codeph>xmlparse</codeph>
```
Comment (dyozie) [inline gpdb-doc/dita/ref_guide/config_params/guc-list.xml]: Change "to <codeph>xml</codeph> data type" -> "to the <codeph>xml</codeph> data type"

File: gpdb-doc/dita/ref_guide/config_params/guc-list.xml [hunk 4]
```diff
@@ -9082,4 +9086,77 @@ $ gpconfig -c pgcrypto.fips -v on --masteronly</codeblock></p>
...
+          function with the <codeph>document</codeph> keyword or change the
```
Comment (dyozie) [inline gpdb-doc/dita/ref_guide/config_params/guc-list.xml]: Change "keywor or" -> "keyword, or"

File: gpdb-doc/dita/ref_guide/config_params/guc-list.xml [hunk 5]
```diff
@@ -9082,4 +9086,77 @@ $ gpconfig -c pgcrypto.fips -v on --masteronly</codeblock></p>
...
+            <codeph>xmloption</codeph> value to <codeph>document</codeph>.</p></note>
+      <p>The parameter can be set for a database system, an individual database, or a session. The
+        SQL command to set this option for a session is also available in Greenplum
+        Database<codeblock>SET XML OPTION { DOCUMENT | CONTENT }</codeblock></p>
```
Comment (dyozie) [inline gpdb-doc/dita/ref_guide/config_params/guc-list.xml]: Missing space at "Database\<codeblock\>SET"

File: gpdb-doc/dita/ref_guide/sql_commands/COPY.xml [hunk 6]
```diff
@@ -284,6 +284,12 @@ COPY {table [(<varname>column</varname> [, ...])] | (<varname>query</varname>)}
         portability to other Greenplum Database installations that might use non-default
           <codeph>DateStyle</codeph> settings, <codeph>DateStyle</codeph> should be set to ISO
         before using <codeph>COPY TO</codeph>.</p>
+      <p>When copying XML data from a file in text mode, the server configuration parameter
+            <codeph><xref href="../config_params/guc-list.xml#xmloption"
+        >xmloption</xref></codeph>affects the validation of the XML data that is copied. If the
```
Comment (dyozie) [inline gpdb-doc/dita/ref_guide/sql_commands/COPY.xml]: Missing space in "\</codeph\>affects"

**Comment by mkiyama** [comment]
I think I got all the typos.

**Pull request merged**